source/jormungandr/jormungandr/interfaces/v1/Places.py

```diff
@@ -198,7 +199,10 @@ def get(self, region=None, lon=None, lat=None):
             response = i_manager.dispatch(args, "places", instance_name=self.region)
         else:
             available_instances = get_all_available_instances(user)
-            autocomplete = global_autocomplete.get('bragi')
+            # If parameter '_autocomplete' is absent use 'bragi' as default value
+            if args["_autocomplete"] is None:
```

Comment:
maybe you can set a default value in that argument?

Reply:
* Here, parameter exist but without any value : args["_autocomplete"] is None
* If we don't use the private parameter '_autocomplete' it is present without any value.
* In this case we cannot init a default value.